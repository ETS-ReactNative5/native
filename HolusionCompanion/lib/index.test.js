
//Simply check if index can be imported
import * as m from "./index";

describe("main module", function(){
  it("has expected properties", function(){
    expect(Object.keys(m)).toMatchSnapshot();
  });
})