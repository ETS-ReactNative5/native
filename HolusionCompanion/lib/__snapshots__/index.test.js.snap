// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main module has expected properties 1`] = `
Array [
  "convert",
  "netScan",
  "screens",
  "selectors",
  "time",
  "useAutoPlay",
  "useWatch",
  "wrapAutoPlay",
  "BgIcon",
  "Bytes",
  "ImageCard",
  "NetworkIcon",
  "ObjectLink",
  "PlayPause",
  "PrevNext",
  "Redirect",
  "SpriteCube",
  "StatusIcon",
  "BaseView",
  "WikiView",
  "AppConfiguration",
  "AppState",
  "Controller",
  "ErrorHandler",
  "InitialLoadWrapper",
  "ListObjects",
  "RequiredLoadOnceWrapper",
  "RequiredLoadWrapper",
  "SafeThemeProvider",
  "ThemeProvider",
  "ifRequiredLoaded",
  "withErrorHandler",
]
`;
